File: 7. C++/Desempeño_Java/CalculosNumericos.java
import java.io.*;
import java.math.*;
//import java.lang.ClassNotFoundException;
import java.util.Scanner;

public class CalculosNumericos
{
	public static void main(String[] args){

		Scanner t = new Scanner(System.in);
		System.out.println("Ingresa N:");
		double MAX = t.nextInt();
		double i=0;
		double seno, coseno, tangente, logaritmo, raizCuad;
		seno=coseno=tangente=logaritmo=raizCuad=0;
		//double MAX=300000;//1
		//double MAX=30000000;//2
		//double MAX=300000000;//3

		while(i<MAX)
		{
			seno+=Math.sin(i);
			coseno+=Math.cos(i);
			tangente+=Math.tan(i);
			logaritmo+=Math.log10(i+1);
			raizCuad+=Math.sqrt(i);
			i++;
		}
		System.out.println("Seno: " + seno + "\nCoseno: " + coseno + "\nTangente: " + tangente + "\nLogaritmo: " + logaritmo + "\nRaíz Cuadrada: " + raizCuad);
	}
}

File: 7. C++/Programa7-3/CalculosNumericos.cpp
#include <iostream>
#include <cmath>

using namespace std;

void function(){
    unsigned long long MAX;
    cin>>MAX;
    double i = 0, seno, coseno, tangente, logaritmo, raizCuadrada;
    seno = coseno = tangente = logaritmo = raizCuadrada = 0;
    while(i < MAX){
        seno += sin(i);
        coseno += cos(i);
        tangente += log(i);
        raizCuadrada += sqrt(i);
        i++;
    }
}

int main(int argc, char const *argv[])
{
    function();
    return 0;
}
